src/drive/web/modules/upload/index.js

```diff
@@ -99,6 +100,11 @@ export const processNextFile = (
       'Upload module needs a cozy-client instance to work. This instance should be made available by using the extraArgument function of redux-thunk'
     )
   }
+
+  if (!CozyFile.cozyClient) {
+    CozyFile.registerClient(client)
+  }
+
```

Comment:
Seems a bit weird to register the client here in a middle of this function. No?

Reply:
moved after cozy client initializations